src/views/ProductView.vue

```diff
@@ -482,6 +478,20 @@ const client = useClient()
  */
 const route = useRoute()
 
+watch(
+    () => route.params.productid,
+    () => {
+        setAmountToNull()
+    }
+)
+
+watch(
```

Comment:
just fyi, you can also make a single watcher watch multiple things

Reply:
That's good to know, thanks!